tpot/export_utils.py

```diff
@@ -21,6 +21,13 @@
 
 import deap
 
+try:
+    import imblearn
+
+    pipeline_module = "imblearn"
```

Comment:
This is not the proper way to determine if a pipeline includes imblearn's resampler. I think you can check `op.import_hash` in this [line](https://github.com/EpistasisLab/tpot/blob/v0.11.6.post1/tpot/export_utils.py#L203) to determine if the pipeline includes a resampler from 'imblearn'. If so, put this [line](https://github.com/EpistasisLab/tpot/blob/v0.11.6.post1/tpot/export_utils.py#L200) after the check and then use imblearn's make_pipeline instead sklearn's one. It may require one more argument for the `_starting_imports ` function.

Reply:
It should work as a generalised way of checking to see if they have imblearn and if so using that pipeline, I will change to only use imblearn on requriement in pipeline though. 

In `import_relations` are the values just strings of the library names?